File: DailyDataServer/src/main/java/com/pseandroid2/dailydataserver/onlineproject/requestParameters/projectParticipantsController/RemoveUserParameter.java
package com.pseandroid2.dailydataserver.onlineproject.requestParameters.projectParticipantsController;

import com.pseandroid2.dailydataserver.RequestParameter;

/**
 * #TODO Testen, JavaDoc
 */
public class RemoveUserParameter extends RequestParameter {
    private String userToRemove;

    /**
     *
     * @param token
     * @param userToRemove the user, which should be removed.
     */
    public RemoveUserParameter(String token, String userToRemove) {
        super(token);
        this.userToRemove = userToRemove;
    }

    public String getUserToRemove() {
        return userToRemove;
    }

    public void setUserToRemove(String userToRemove) {
        this.userToRemove = userToRemove;
    }

    @Override
    public String toString() {
        return "RemoveUserParameter{" +
                "userToRemove='" + userToRemove + '\'' +
                "} " + super.toString();
    }
}

File: DailyData/app/src/main/java/com/pseandroid2/dailydata/remoteDataSource/serverConnection/serverParameter/RemoveUserParameter.kt
package com.pseandroid2.dailydata.remoteDataSource.serverConnection.serverParameter

/**
 * @param token: The authentication token
 * @param userToRemove: The UserID of the user that should be removed from the project
 * @param projectID: The ID of the Project from which the user should be removed
 */
data class RemoveUserParameter(val userToRemove: String, val projectID: Long)
